Reworked Stuff, Added Stuff

SteveMutter opened the pull request:
Reworked the Camera Postitioning

Reworked cameras, removed unnecessary information and cameras, added OV9732, HBVCAM.

Reworked example setups

Reworked hardware requirments, added Cameras, added illumination

Deleted Recommend Ubuntu Version

Added some Pictures to static

SteveMutter committed:
Reworked Stuff, Added Stuff

Reworked the Camera Postitioning

Reworked cameras, removed unnecessary information and cameras, added OV9732, HBVCAM.

Reworked example setups

Reworked hardware requirments, added Cameras, added illumination

Deleted Recommend Ubuntu Version

Added some Pictures to static

SteveMutter committed:
Rewrite IP Adress Part in First Startup section

Reworked the IP Adress section in First Startup section for better understanding

tnolle commented on content/en/getting-started/cameras.md:
```diff
@@ -38,14 +38,12 @@ Here are a few examples from the community that are already being used by our us
-| 1           | 5          | ![img](/camera-comp/images/aode_1080.png)    | AODE       | 1080 HD USB 2.0 | 10          | 90      | Yes         | Yes          | Auto     | Yes       | massive + stable                                                                                                                                           | still too mutch distortion / too expensive / USB problems ( ...
+| ![img](/camera-comp/images/elp_otg.png)      | ELP OTG HD 5mp        | 100     | None        | None         | Manual   | No        |
+| ![img](/camera-comp/images/mortek_mc920.png) | MOERTEK    | 85      | Nearly none | No           | Auto     | Yes       |
+| ![img](/camera-comp/images/HBV.png)   | HBVCAM-W2017-V22    | 110     | Little         | No           | Auto     |No       |                                             |
```
I am a bit confused with the images here. The image called `HBV.png` is not the HBVCAM-W2017-V22, the OV9732 below is. I would just remove the `HBV.png` image. That camera will not work because it has the wrong lens on it.

SteveMutter replied:
I will remove the HBV, but then we should also remove it from the guidelines (Discord / Marco)

tnolle commented on content/en/getting-started/camera-positioning.md:
```diff
@@ -7,44 +7,34 @@ weight: 3
 
 ![How to mount cameras](/camera-mount/images/camera_mount_side.png)
 
+![Cameramount](/camera-mount/images/Dartboard_Plasma.jpg)
```
This picture shows the Design by Robbel90, not the one that Autodarts has in its docs under 3d-printing. Not sure if we want to intermix Robbel90's design with Lars design within the docs. This might lead to confusion. However, what we should do, is to include Robbel90's design as a link and perhaps even say that his design is recommended (for 10 Euros) at the top of 3d-printing? What do you think?

SteveMutter replied:
Done

SteveMutter committed:
Update Cameras.md

Removed HBV from the list of Cameras

SteveMutter committed:
Remove Cameras, Added Robbel to 3d Print page

Removed old Pictures from Cameras, Removed Robbels Example Image from camera-postitioning.md

Added Note for Robbel on the 3d print page with a link to his Designs on Cult

SteveMutter committed:
Update uvc-hack.md

Added short description

SteveMutter committed:
Update installation.md

Added Download Link to Github Releases Page / Will be Obsolete if new AD Desktop is Released

SteveMutter committed:
Changed UVC Hack

Copied UVC Hack in Linux Installation cause of relevance to only Linux systems... Set UVC Hack Draft to True

tnolle commented:
Looks good. Thank you!
